test/test727.js

```diff
@@ -0,0 +1,18 @@
+if(typeof exports === 'object') {
+	var assert = require("assert");
+	var alasql = require('..');
+	var fs = require("fs");
+}
+
+var test = '727'; // insert test file number

```

Comment:
Makes sense that you gave it the ID of the PR - but please give it 612 as thats the next number in the row of test files